blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeImpl.java

```diff
@@ -183,13 +202,43 @@ private WorkflowRun getRun() {
 
     @Override
     public boolean isRestartable() {
-        RestartDeclarativePipelineAction restartDeclarativePipelineAction =
-            getRun().getAction( RestartDeclarativePipelineAction.class );
-        if (restartDeclarativePipelineAction != null) {
-            List<String> restartableStages = restartDeclarativePipelineAction.getRestartableStages();
-            if (restartableStages != null) {
-                return restartableStages.contains(this.getDisplayName())
-                    && this.getStateObj() == BlueRun.BlueRunState.FINISHED;
+        return restartable;
+    }
+
+    /**
+     * @return the Pipeline execution
+     * @todo: This can be removed once https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/596 is available.
```

Comment:
https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/596 can be picked up as an incremental build so long as this downstream PR is in draft.

Reply:
target branch is 1.25.5.x so we will not upgrade. sounds like a leftover of the master commit